xray/plot/plot.py

```diff
@@ -384,7 +435,11 @@ def _plot2d(plotfunc):
     Parameters
     ----------
     darray : DataArray
-        must be 2 dimensional.
+        Must be 2 dimensional
+    x : string, optional
+        Coordinate for x axis. If None use darray.dims[1]
```

Comment:
Surprisingly no. Not if we want to be consistent with `plt.imshow`. Consider:

```
In [1]: a = np.arange(100).reshape(10, 10)

In [2]: b = xray.DataArray(a)

In [3]: b
Out[3]:
<xray.DataArray (dim_0: 10, dim_1: 10)>
array([[ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9],
       [10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
       [20, 21, 22, 23, 24, 25, 26, 27, 28, 29],
       [30, 31, 32, 33, 34, 35, 36, 37, 38, 39],
       [40, 41, 42, 43, 44, 45, 46, 47, 48, 49],
       [50, 51, 52, 53, 54, 55, 56, 57, 58, 59],
       [60, 61, 62, 63, 64, 65, 66, 67, 68, 69],
       [70, 71, 72, 73, 74, 75, 76, 77, 78, 79],
       [80, 81, 82, 83, 84, 85, 86, 87, 88, 89],
       [90, 91, 92, 93, 94, 95, 96, 97, 98, 99]])
Coordinates:
  * dim_0    (dim_0) int64 0 1 2 3 4 5 6 7 8 9
  * dim_1    (dim_1) int64 0 1 2 3 4 5 6 7 8 9

In [4]: b.plot()
Out[4]: <matplotlib.image.AxesImage at 0x10f048f10>

In [5]: b.dims
Out[5]: ('dim_0', 'dim_1')
```

![image](https://cloud.githubusercontent.com/assets/5356122/9688933/2f61522e-52e7-11e5-8a56-571f912e999b.png)